### PR TITLE
Prepare changelog for 0.11

### PR DESCRIPTION
**Pull request by kolloch**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -51,18 +99,220 @@ deliver information you explicitly requested by default, even though we will
...
+When Marathon has no current use for an offer, it will decline the offer for a configurable period. This period is
+configurable. A short duration might lead to resource starvation for other frameworks if you run many frameworks
+in your cluster. You should only need to reduce it if you use `--disable_revive_offers_for_new_apps`.
+
+* `--decline_offer_duration` (Default: 120 seconds) The duration (milliseconds) for which to decline offers by default
+
+#### New task launching tuning parameters
+
+To prevent overloading Mesos itself, you can now restrict how many tasks Marathon launches per time interval.
+By default, we allow 1000 unconfirmed task launches every 30 seconds. In addition, Marathon launches
```
Comment (aquamatthias) [inline changelog.md]: The sentence is correct, but the small word `unconfirmed` should be explained more detailed.

File: changelog.md [hunk 2]
```diff
@@ -51,18 +99,220 @@ deliver information you explicitly requested by default, even though we will
...
+-------------|----------------
+169 | Felix Gertz
+68 |  philipnrmn
+50 | Pierluigi Cau
+5 | Sp3c1
+4 | Kamil Warguła
+4 | Daniel Fuentes
+2 | Philip Norman
+
+Generated by `git shortlog -s -n --no-merges v0.10.0..v0.11.0` for the marathon-ui repository
```
Comment (aquamatthias) [inline changelog.md]: the marathon project will be added I guess.

Comment (kolloch) [inline changelog.md]: yepp, have to wait for the PR.

**Comment by kolloch** [comment]
I haven't dealt with the comments yet but rebased it on the predecessor PR.

**Comment by kolloch** [comment]
Alright, I took care of Matthias suggestions's.

Contributor stats come when we are finished with all tasks for 0.11.

File: changelog.md [hunk 3]
```diff
@@ -108,18 +99,221 @@ deliver information you explicitly requested by default, even though we will
 allow some compatibility configuration option. This improves performance by not
 returning information that you might not need.
 
+#### Smarter resource offer handling
+
+When Marathon needed to launch tasks for multiple apps, it would first launch all needed tasks for the first one
+and then proceeding with the next app. One big deployment could block all others. In this new version, Marathon
```
Comment (philipnrmn) [inline changelog.md]: s/proceeding with/proceed to/

File: changelog.md [hunk 4]
```diff
@@ -108,18 +99,221 @@ deliver information you explicitly requested by default, even though we will
...
+will spread the offers among all apps that currently need to launch tasks.
+
+When Marathon detects new tasks that it has to launch, then it will now explicitly request new offers
```
Comment (philipnrmn) [inline changelog.md]: s/then//

File: changelog.md [hunk 5]
```diff
@@ -108,18 +99,221 @@ deliver information you explicitly requested by default, even though we will
...
+with the `reviveOffers` Mesos scheduler API call. This should result in more speedy task launches. If
+for any reason, you dislike this behavior, you can disable it with `--disable_revive_offers_for_new_apps`
```
Comment (philipnrmn) [inline changelog.md]: s/reason,/reason/

File: changelog.md [hunk 6]
```diff
@@ -108,18 +99,221 @@ deliver information you explicitly requested by default, even though we will
...
+(not recommended).
+
+When Marathon has no current use for an offer, it will decline the offer for a configurable period. This period is
+configurable. A short duration might lead to resource starvation for other frameworks if you run many frameworks
```
Comment (philipnrmn) [inline changelog.md]: `...for a configurable period. This period is configurable`
This duplication can be removed.

File: changelog.md [hunk 7]
```diff
@@ -108,18 +99,221 @@ deliver information you explicitly requested by default, even though we will
...
+[2015-08-04 20:17:27,986] INFO Offer [20150804-173037-16777343-5050-4340-O110]. Not all basic resources satisfied: cpu NOT SATISFIED (30.0 > 8.0), disk SATISFIED (0.0 <= 0.0), mem SATISFIED (16.0 <= 15360.0)
+[2015-08-04 20:17:27,989] INFO Offer [20150804-173037-16777343-5050-4340-O110]. Insufficient for [/test] (need cpus=30.0, mem=16.0, disk=0.0, ports=(1 dynamic), available in offer:
+...
+```
+
+#### Immediately store changed apps
+
+In the past, Marathon would update the data for the group endpoint when it accepted a new deployment request
+like a change an app configuration. Marathon would only update the data in the app endpoint when it started
+the related deployment step. All in all, that was confusing for our users and thus we will now update both
```
Comment (philipnrmn) [inline changelog.md]: `All in all` is a slightly odd switch of tone. How about
`We found that this was confusing...`

File: changelog.md [hunk 8]
```diff
@@ -108,18 +99,221 @@ deliver information you explicitly requested by default, even though we will
...
+Jetty 9 has a completely overhauled I/O layer, Servlet API 3.0, SPDY/3 and WebSocket support.
+
+#### Improved SSE handling
+
+As part of the Jetty 9 update, the SSE support for `/v2/events` has been improved.
+The event name `event: event-name` is added to every `data: json` entry for easier filtering and handling. 
+
 #### Play JSON everywhere
 
 The finished our transition from Jackson JSON serialization to Play JSON. Play JSON provides a type safe
```
Comment (philipnrmn) [inline changelog.md]: `The finished our transition` doesn't make sense

File: changelog.md [hunk 9]
```diff
@@ -108,18 +99,221 @@ deliver information you explicitly requested by default, even though we will
...
+
+Commits | Contributor
+-------------|----------------
+169 | Felix Gertz
+68 |  philipnrmn
+50 | Pierluigi Cau
+5 | Sp3c1
+4 | Kamil Warguła
+4 | Daniel Fuentes
+2 | Philip Norman
```
Comment (philipnrmn) [inline changelog.md]: Can we aggregate Philip Norman and philipnrmn please?

Comment (philipnrmn) [inline changelog.md]: Asking for a friend ;)

File: changelog.md (hunk 3 above)
Comment (kolloch) [inline changelog.md]: jepp

File: changelog.md (hunk 4 above)
Comment (kolloch) [inline changelog.md]: thx

File: changelog.md (hunk 6 above)
Comment (kolloch) [inline changelog.md]: indeed

File: changelog.md (hunk 7 above)
Comment (kolloch) [inline changelog.md]: :+1:

File: changelog.md (hunk 8 above)
Comment (kolloch) [inline changelog.md]: s/The/We/

File: changelog.md (hunk 9 above)
Comment (kolloch) [inline changelog.md]: you might want to do it in your copy of the changelog, too

**Comment by kolloch** [comment]
Thanks, @philipnrmn. I added a commit with fixes. I can squash it if you're done with reviewing the changes.

**Comment by philipnrmn** [comment]
LGTM. Squash + :ship:

**Comment by kolloch** [comment]
Squashed!!

**Pull request merged**